platform/ios/src/MGLMapView.mm

```diff
@@ -2630,6 +2630,18 @@ - (void)setShowsScale:(BOOL)showsScale
     }
 }
 
+- (void)setPrefetchesTiles:(BOOL)prefetchesTiles{
+    if(!prefetchesTiles)
+    {
+        _mbglMap->setPrefetchZoomDelta(0);
+    }
+    else
+    {
+        //Reset to default value.
+        _mbglMap->setPrefetchZoomDelta(4);
```

Comment:
Per https://github.com/mapbox/mapbox-gl-native/issues/9438#issuecomment-314246642, use the constant `DEFAULT_PREFETCH_ZOOM_DELTA` instead of hard-coding the literal number 4. Also, the if/else statement can be written more succinctly as a ternary expression: `setPrefetchZoomDelta(prefetchesTiles ? DEFAULT_PREFETCH_ZOOM_DELTA : 0)`.